cls/SourceControl/Git/Extension.cls

```diff
@@ -59,12 +59,17 @@ Method UserAction(Type As %Integer, Name As %String, InternalName As %String, Se
             // if item is being edited in a different namespace, opening it will display an alert. Editing in this namespace will remove the alert.
             set filename = ##class(SourceControl.Git.Utils).FullExternalName(.InternalName)
             do ##class(SourceControl.Git.Change).GetUncommitted(filename,.tAction)
+            do ..GetStatus(.InternalName, .isInSourceControl, .isEditable,.isCheckedOut,.userCheckedOut)
             if '$data(tAction) {
                 set user = "", inNamespace = ""
                 if ##class(SourceControl.Git.Utils).InstanceWideUncommittedCheck(InternalName, .user, .inNamespace) {
                     set Target = "Warning: Item " _ InternalName _ " is currently being modified by " _ user _ " in namespace " _ inNamespace _ "."
                     write !, Target
                     set Action = 6
+                } elseif 'isEditable || ##class(SourceControl.Git.Utils).Locked() {
```

Comment:
I think this should actually come before the instance-wide uncommitted check. "You can't edit this here" is more important to say than "someone else is editing this somewhere else"